src/Command/App/NewCommand.php

```diff
@@ -52,17 +53,26 @@ protected function execute(InputInterface $input, OutputInterface $output): int
       $dir = '/home/ide/project';
     }
     else {
-      $dir = Path::makeAbsolute('drupal', getcwd());
+      $dir = Path::makeAbsolute($project, getcwd());
     }
 
     $output->writeln('<info>Creating project. This may take a few minutes.</info>');
-    $this->localMachineHelper->checkRequiredBinariesExist(['composer']);
-    $this->createProject($project, $dir);
+    $success_message = "<info>New 💧 Drupal project created in $dir. 🎉";
```

Comment:
Move this into the conditional for consistency

Reply:
PR updated!